policyengine_us/variables/gov/states/va/tax/income/deductions/itemized/child_dependent.py

```diff
@@ -13,7 +13,7 @@ def formula(tax_unit, period, parameters):
         amount = parameters(
             period
         ).gov.states.va.tax.income.deductions.itemized.child_dependent
-        if tax_unit("tax_unit_dependents", period) < 2:
-            return amount
-        else:
-            return amount * 2
+
+        return np.where(
```

Comment:
We have an alias for `where`, so you don't need to import numpy
```suggestion
        return where(
```